packages/calling-stateful-client/src/CallAgentDeclarative.test.ts

```diff
@@ -259,8 +259,8 @@ describe('declarative call agent', () => {
 
     expect(Object.keys(context.getState().calls).length).toBe(0);
     expect(Object.keys(context.getState().callsEnded).length).toBe(1);
-    expect(context.getState().callsEnded[mockCallId].callEndReason?.code).toBe(1);
-    expect(context.getState().callsEnded[mockCallId].endTime).toBeTruthy();
+    expect(context.getState().callsEnded[mockCallId]?.callEndReason?.code).toBe(1);
```

Comment:
The `?` is added because `context.getState().callsEnded[mockCallId]` now correctly returns `Call | undefined` (importantly the `| undefined` is now added to typescripts interpretation of the return type because the call is accessed via `object[key]` where `key` isn't guaranteed to exist)